src/test/java/usecasetesting/ClientRegisterUseCaseTest.java

```diff
@@ -0,0 +1,129 @@
+package usecasetesting;
+
+import adapter.controller.ControlContainer;
+import businessrule.gateway.ClientGateway;
+import businessrule.gateway.UserGatewayFactory;
+import businessrule.inputboundary.ClientRegisterInputBoundary;
+import businessrule.inputboundary.UserLoginInputBoundary;
+import businessrule.outputboundary.HomePageOutputBoundary;
+import businessrule.outputboundary.RegisterOutputBoundary;
+import businessrule.requestmodel.ClientRegisterRequestModel;
+import businessrule.responsemodel.HomePageResponseModel;
+import businessrule.responsemodel.RegisterResponseModel;
+import businessrule.usecase.ClientRegisterInteractor;
+import businessrule.usecase.UserLoginInteractor;
+import driver.database.ClientRepository;
+import entity.Client;
+import entity.ClientFactory;
+import org.junit.BeforeClass;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLOutput;
+import java.time.LocalDate;
+
+import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.*;
+;
+
+public class ClientRegisterUseCaseTest {
+    final static String PASSWORD = "abcdefghi";
+    private ClientFactory clientFactory;
+    private ClientGateway clientGateway;
+    private RegisterOutputBoundary registerOutputBoundary;
+    private ClientRegisterInputBoundary clientRegisterInputBoundary;
+    public void setUpClientRegisterUseCase(){
+        clientGateway = new ClientRepository();
+        clientFactory = new ClientFactory();
+        clientGateway.deleteAllUser();
+        registerOutputBoundary = new RegisterOutputBoundary() {
+            @Override
+            public void setControlContainer(ControlContainer controlContainer) {
+
+            }
+
+            @Override
+            public RegisterResponseModel prepareSuccess(String msg) {
+                System.out.println(msg);
+                System.out.println("Registration is approved.");
+                return null;
+            }
+
+            @Override
+            public RegisterResponseModel prepareFail(String msg) {
+                System.out.println(msg);
+                return null;
+            }
+        };
+
+        clientRegisterInputBoundary = new ClientRegisterInteractor(clientGateway, registerOutputBoundary, clientFactory);
+
+    }
+    @Test
+    public void TestSuccessfulRegistration(){
+        setUpClientRegisterUseCase();
+
+        ClientRegisterRequestModel inputData = new ClientRegisterRequestModel("joseph", "test@gmail.com", PASSWORD, PASSWORD, "test state", "21000", "test ethinicity", 18, "male", "single", 1, 1000.0F);
```

Comment:
I believe it is better practice to set these values as instance variables instead of slapping the raw value straight into the initialization.